solarforecastarbiter/reference_forecasts/forecast.py

```diff
@@ -0,0 +1,205 @@
+"""
+Functions for forecasting.
+"""
+
+from solarforecastarbiter import pvmodel
+
+# some functions originally implemented in pvlib-python.
+# find pvlib-python license in licenses directory.
+
+
+def cloud_cover_to_ghi_linear(cloud_cover, ghi_clear, offset=35):
+    """
+    Convert cloud cover to GHI using a linear relationship.
+    0% cloud cover returns ghi_clear.
+    100% cloud cover returns offset*ghi_clear.
+
+    Parameters
+    ----------
+    cloud_cover: numeric
+        Cloud cover in %.
+    ghi_clear: numeric
+        GHI under clear sky conditions.
+    offset: numeric, default 35
+        Determines the minimum GHI.
+
+    Returns
+    -------
+    ghi: numeric
+        Estimated GHI.
+
+    References
+    ----------
+    Larson et. al. "Day-ahead forecasting of solar power output from
+    photovoltaic plants in the American Southwest" Renewable Energy
+    91, 11-20 (2016).
+    """
+
+    offset = offset / 100.
+    cloud_cover = cloud_cover / 100.
+    ghi = (offset + (1 - offset) * (1 - cloud_cover)) * ghi_clear
+    return ghi
+
+
+def cloud_cover_to_irradiance_clearsky_scaling(cloud_cover, ghi_clear,
```

Comment:
Seems like this function and the following would be simpler as one function, that had solar position as kwargs, and calculates solpos if defaulted to None